services/workflows-service/src/alert/alert.service.intg.test.ts

```diff
@@ -182,6 +184,8 @@ describe('AlertService', () => {
               enabled: true,
             },
             project,
+            undefined,
+            { crossEnvKey: 'TEST' },
```

Comment:
you could just use `generateAlertDefinition` function which generate automatic prefix from project name :)